test/units/testsuite-50.sh

```diff
@@ -540,6 +540,17 @@ systemctl stop test-root-ephemeral
 timeout 10 bash -c 'while ! test -z "$(ls -A /var/lib/systemd/ephemeral-trees)"; do sleep .5; done'
 test ! -f /tmp/img/abc
 
+# Test that systemd-sysext reloads the daemon.
+mkdir -p /var/lib/extensions/
+ln -s /usr/share/app-nodistro.raw /var/lib/extensions/app-nodistro.raw
+systemd-sysext merge
```

Comment:
```suggestion
systemd-sysext merge
for RETRY in $(seq 60); do
  (
  if [ "${RETRY}" -lt 60 ]; then
    set +e
  fi
  systemctl status foo.service | grep -q -P 'echo\[[0-9]+\]: foo'
  sleep 0.5
  )
done
```
I think it would be good to check that the service is started which is the goal here.